data/templates/css/wikimedia.scss

```diff
@@ -1,5 +1,4 @@
 @import "project-setup";
-@import "compass";
```

Comment:
@tchx84 `pie-clearfix` is a cross-browser fix for extending the bottom of the nesting element: http://compass-style.org/reference/compass/utilities/general/clearfix/